lib/src/main/java/com/auth0/jwt/interfaces/Verification.java

```diff
@@ -176,7 +177,8 @@ default Verification withAnyOfAudience(String... audience) {
     Verification withClaim(String name, String value) throws IllegalArgumentException;
 
     /**
-     * Require a specific Claim value.
+     * Require a specific Claim value. Note that date-time claims are serialized as seconds since the epoch; when verifying
+     * date-time claim value, any time units more granular than seconds will be discarded.
```

Comment:
Could we rather say "not be considered/included" instead of "discarded"? Since I already have the context of the fix I am able to understand this easily. Not sure for the first time reader. But I could be wrong here

Reply:
Updated the PR to use the terminology "will not be considered"